packages/gitbeaker-requester-utils/src/RequesterUtils.ts

```diff
@@ -34,17 +35,18 @@ export function defaultRequest(
   service: Service,
   { body, query, sudo, method = 'get' }: DefaultRequestOptions = {},
 ): Record<string, string | number | FormData | Agent | Record<string, string | string[] | Agent>> {
-  const { headers } = service;
+  const { headers, additionalBody } = service;
   let agent;
   let bod;
 
   if (sudo) headers.sudo = sudo;
 
   // FIXME: Not the best comparison, but...it will have to do for now.
   if (typeof body === 'object' && body.constructor.name !== 'FormData') {
-    bod = JSON.stringify(decamelizeKeys(body));
+    bod = JSON.stringify(decamelizeKeys({ ...body, ...additionalBody }));
     headers['content-type'] = 'application/json';
   } else {
+    /** TODO - what do I do here with the additionalBody? */
```

Comment:
Good question, how does these additional body arguments function with non JSON bodies?

Reply:
Hehe, ikr?

I though perhaps we should

```ts
      bod = JSON.stringify({ ...JSON.parse(body), ...additionalBody });
```

? Or would this not work?

Reply:
Not sure why you are parsing the body here, since its already an object, but otherwise it should be fine

Reply:
https://github.com/jdalrymple/gitbeaker/pull/1106#discussion_r497757409